--- a/src/renderer/views/pages/cider-playlist.ejs
+++ b/src/renderer/views/pages/cider-playlist.ejs
@@ -115,7 +115,7 @@
                                     {{ (!inLibrary) ? app.getLz('action.addToLibrary') :
                                     app.getLz("action.removeFromLibrary") }}
                                 </button>
-                                <button class="md-btn md-btn-icon" style="min-width: 180px;" v-if="confirm==true"
+                                <button class="md-btn md-btn-icon md-btn-confirm" style="min-width: 180px;" v-if="confirm==true"
                                         @click="(!inLibrary) ? addToLibrary(data.attributes.playParams.id.toString()) : removeFromLibrary(data.attributes.playParams.id.toString()) ">
                                     <img :class="(!inLibrary) ? 'md-ico-add' : 'md-ico-remove'">
                                     {{app.getLz('term.confirm')}}
@@ -185,7 +185,7 @@
                                 {{ (!inLibrary) ? app.getLz('action.addToLibrary') :
                                 app.getLz("action.removeFromLibrary") }}
                             </button>
-                            <button class="md-btn md-btn-icon" style="min-width: 180px;" v-if="confirm==true"
+                            <button class="md-btn md-btn-icon md-btn-confirm" style="min-width: 180px;" v-if="confirm==true"
                                     @click="(!inLibrary) ? addToLibrary(data.attributes.playParams.id.toString()) : removeFromLibrary(data.attributes.playParams.id.toString()) ">
                                 <img :class="(!inLibrary) ? 'md-ico-add' : 'md-ico-remove'">
                                 {{app.getLz('term.confirm')}}
